R/bambu.R

```diff
@@ -127,6 +127,10 @@ bambu <- function(reads = NULL, rcFile = NULL, rcOutDir = NULL,
             readClass.outputDir = rcOutDir, genomeSequence = genome)
     if(!is.null(reads)) genomeSequence <- checkInputSequence(genome)
     isoreParameters <- setIsoreParameters(isoreParameters = opt.discovery)
+
+    #below line is to be compatible with earlier version of running bambu
+    if(!is.null(isoreParameters$max.txNDR)) NDR = isoreParameters$max.txNDR
+
```

Comment:
it will be a bit confusing to users I guess if we have two places to set the NDR threshold, I would suggest to just move NDR out of the isoreParameters. It is not exactly a new feature, right? We probably also need to update the documentation for this as well